app/models/licence.rb

```diff
@@ -0,0 +1,25 @@
+require 'ostruct'
+
+class Licence
+  LicenceInfo = Struct.new(:title, :url)
+
+  LICENCES = {
```

Comment:
How about we sort this list alphabetically to make the code a bit easier to scan?